include/sqlpp11/detail/float_safe_ostringstream.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (c) 2023, Roland Bock
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without modification,
+ * are permitted provided that the following conditions are met:
+ *
+ *   Redistributions of source code must retain the above copyright notice, this
+ *   list of conditions and the following disclaimer.
+ *
+ *   Redistributions in binary form must reproduce the above copyright notice, this
+ *   list of conditions and the following disclaimer in the documentation and/or
+ *   other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR
+ * ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+ * (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ * LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ * ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+ * SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#ifndef SQLPP11_COLUMN_TUPLE_FLOAT_SAFE_OSTRINGSTREAM_H
+#define SQLPP11_COLUMN_TUPLE_FLOAT_SAFE_OSTRINGSTREAM_H
+
+#include <limits>
+#include <sstream>
+#include <type_traits>
+#include <utility>
+
+#include <sqlpp11/detail/enable_if.h>
+#include <sqlpp11/detail/remove_cvref.h>
+
+namespace sqlpp
+{
+  namespace detail
+  {
+    template<typename T, typename = void>
+    struct float_safe_ostringstream_implementation
+    {
+      template<typename U>
+      void operator() (std::ostringstream& os, U&& x) const
+      {
+        os << std::forward<U> (x);
```

Comment:
nit: This space looks odd to me. Did you run this through clang-format?

Reply:
Yep, sorry, I tried to follow existing style but apparently failed in quite a few locations. Running through clang-format does change quite a lot of existing code, so I didn't think it was the current style. I'll run the files I touched through it and pick only those changes that also touch the lines I touched in the existing files.

Reply:
OK, I should look into this then. The config is rather old by now and clang-format has been under heavy development :-)